config/initializers/country_config.rb

```diff
@@ -45,7 +45,7 @@ class CountryConfig
         monthly_registrations: "tI9g0mtCzOv"
       },
       enabled_diabetes_population_coverage: true,
-      appointment_reminders_channel: "Messaging::Twilio::ReminderSms"
+      appointment_reminders_channel: "Messaging::AlphaSms::Sms"
```

Comment:
Why no distinction between OTP and Reminder SMS in this channel? Is it because there aren't any "multiple senders" to leverage like we had in Twilio?